src/models/StyleSheet.js

```diff
@@ -1,35 +1,142 @@
 // @flow
+import BrowserStyleSheet from './BrowserStyleSheet'
+import ServerStyleSheet from './ServerStyleSheet'
 
-/* Wraps glamor's stylesheet and exports a singleton for the rest
-*  of the app to use. */
+export const SC_ATTR = 'data-styled-components'
+export const LOCAL_ATTR = 'data-styled-components-is-local'
```

Comment:
It doesn't really have to be this short, I'd say. This name is pretty unambiguous